### ISSUE TITLE
New Task: Add EuroPMC Dataset

### ISSUE THREAD

**Issue by abhinav-kashyap-asus** [OpenBioML/chemnlp#32]
I would like to work on this. 
I took this dataset from the awesome list.
What is the priority for this?? Any more information that I need to have in mind before starting to add this dataset?

**Comment by MicPie** [comment]
Hi @abhinav-kashyap-asus 
I guess a first step would be to have a look how the data is organized as it it contains different data from different sources.
We can also discuss this in more detail on our Discord. Just ping me / us there! :-)

**Comment by abhinav-kashyap-asus** [comment]
Thanks for this guys. I just saw that EuroPMC has bulk download of more than 5 million articles which are open access. 
I have to see the exact details of the license information for these articles and whether we can use them. 
As you mentioned in #18, this dataset also some figures available. 

A good first step would be may be to just check the metadata (https://europepmc.org/ftp/pmclitemetadata/) and see if it contains some licensing information. 

I will ping you guys separately on the Discord server for this
